src/components/calculator/calculator.module.css

```diff
@@ -0,0 +1,17 @@
+.calculatorWrapper {
+  display: flex;
+  flex-direction: column;
+  background-color: rgb(184, 180, 180);
+  width: 400px;
+}
+
+.display {
+  padding: 15px;
+  text-align: right;
+  font-size: 20px;
+  color: white;
+}
+
+.row {
+  display: flex;
+}
```

Comment:
- Kindly ensure your app is displaying the required UI and not the CRA default UI 👍🏽 
![Screenshot from 2023-04-26 19-45-31](https://user-images.githubusercontent.com/44624138/234679055-d195ce69-f8ad-4c79-aeb1-9afd7c1056fb.png)